gui/doc/chart.md

```diff
@@ -3,30 +3,268 @@ Displays data sets in a chart or a group of charts.
+      </a></li>
+    <li><a href="../charts/funnel">
+        <span>Funnel charts<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><use xlink:href="#rarrow"/></svg></span>
+        <img src="../charts/home-funnel-d.png"  class="visible-dark" />
+        <img src="../charts/home-funnel-l.png"  class="visible-light" />
+      </a></li>
+</ul>
+
+<h2 class="h2">Maps</h2>
+<ul class="list">
+    <li><a href="../charts/chart-map">
+        <span>Charts on maps<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><use xlink:href="#rarrow"/></svg></span>
+        <img src="../charts/home-chart-map-d.png"  class="visible-dark" />
+        <img src="../charts/home-chart-map-l.png"  class="visible-light" />
+      </a></li>
+    <li><a href="../charts/area-map">
+        <span>Areas on maps<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><use xlink:href="#rarrow"/></svg></span>
+        <img src="../charts/home-area-map-d.png"  class="visible-dark" />
+        <img src="../charts/home-area-map-l.png"  class="visible-light" />
+      </a></li>
+    <li><a href="../charts/density-map">
+        <span>Density maps<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><use xlink:href="#rarrow"/></svg></span>
+        <img src="../charts/home-density-map-d.png"  class="visible-dark" />
+        <img src="../charts/home-density-map-l.png"  class="visible-light" />
+      </a></li>
+    <!-- Choropleth maps - choropleth-map -->
+</ul>
+
+A chart control can hold several traces, that can display individual data sets.<br/>
 To indicate properties for a given trace, you will use the indexed properties
 (using the *property_name[index]* syntax, with the indices starting at index 1) to
 specify which trace you target.
 
 Indexed properties can have a default value (using the *property_name* syntax with
-no index) which is overridden by any specified indexed property.
+no index) which is overridden by any specified indexed property:<br/>
+if `indexed_property` is an indexed property, you can set it to a value, that is
+then applied to all the traces. If you later reference that property with an index, the
+value will apply only for the indicated trace.
+```
+general_value = <some_value>
+specific_value = <some_other_value>
 
-The _data_ property supported types are:
+page = "<|...|chart|...|index_property={general_value}|index_property[2]={specific_value}|...|>"
+```
+creates a chart where *general_value* is used for all the traces, except for the one with index 2.
 
-- pandas Dataframe
-- list of lists
-- numpy series
-- list of pandas dataframes
+An indexed property can also be assigned an array. Then each array value is set to the appropriate
+index, in sequence:
+```
+values = [
+    value1,
+    value2,
+    value3
+]
+    
+page = "<|...|chart|...|index_property={values}|...|>"
+```
+is equivalent to
+```
+page = "<|...|chart|...|index_property[1]={value1}|index_property[2]={value2}|index_property[3]={value3}|...|>"
```

Comment:
<|...|chart|...|index_property={value1}|index_property[1]={value1}|index_property[2]={value2}|index_property[3]={value3}|...|>

Reply:
I'm sorry. I don't get your point...

Reply:
when you specify an array, the first value of the array becomes the default value.
ie if your array is shorter than the number of traces, the extra traces inherit from the 1st value of the array (like they would if the non index property was not an array)

Reply:
Yes.
So I guess your
`index_property={value1}|index_property[1]={value1}`
was meant to be
`index_property={value1}`

Reply:
Which is the same in the end (and might be clearer ?)